amethyst_network/src/network_socket.rs

```diff
@@ -24,18 +24,19 @@ enum InternalSocketEvent<E> {
     Stop,
 }
 
-// If a client sends both a connect event and other events,
-// only the connect event will be considered valid and all others will be lost.
-/// The System managing the network state and connections.
-/// The T generic parameter corresponds to the network event type.
-/// Receives events and filters them.
-/// Received events will be inserted into the NetReceiveBuffer resource.
-/// To send an event, add it to the NetSendBuffer resource.
+/// The System managing the network state from `NetConnections`.
 ///
-/// If both a connection (Connect or Connected) event is received at the same time as another event from the same connection,
-/// only the connection event will be considered and rest will be filtered out.
-// TODO: add Unchecked Event type list. Those events will be let pass the client connected filter (Example: NetEvent::Connect).
-// Current behaviour: hardcoded passthrough of Connect and Connected events.
+/// This system has a few responsibilities.
+///
+/// - Reading all send packets from `NetConnection` and send those over to some remote endpoint.
```

Comment:
"Reading all sent packets from `NetConnection` and sending those over to some remote endpoint."

Reply:
I need to make it 'to send' actually because the packets are queued and need to be sent by the system.

"Reading to send packets from `NetConnection` and sending those over to some remote endpoint."